### DISCUSSION TITLE
Add the back to top button on the right side of github homepage

### DISCUSSION THREAD

**Discussion by guoduhw** [community/community#38152] [Other Feature Feedback, Questions, & Ideas]
On the cool homepage of the new github, if you want to go back to the top immediately when sliding to the lower position, there is no operable button, which will affect the user experience. It is strongly recommended to add the right back to the top button to follow

**Comment by ossanna16** [comment]
Hi there @guoduhw and welcome to our community! Thank you for the feedback 🙂
